packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FLTGoogleMapJSONConversions.m

```diff
@@ -86,6 +113,77 @@ extern GMSMapViewType FGMGetMapViewTypeForPigeonMapType(FGMPlatformMapType type)
                      markerIds:markerIDs];
 }
 
+GMSCameraUpdate *FGMGetCameraUpdateForPigeonCameraUpdate(FGMPlatformCameraUpdate *cameraUpdate) {
+  // See note in messages.dart for why this is so loosely typed.
+  id update = cameraUpdate.cameraUpdate;
+  if ([update isKindOfClass:[FGMPlatformCameraUpdateNewCameraPosition class]]) {
+    return [GMSCameraUpdate
+        setCamera:FGMGetCameraPositionForPigeonCameraPosition(
+                      ((FGMPlatformCameraUpdateNewCameraPosition *)update).cameraPosition)];
+  } else if ([update isKindOfClass:[FGMPlatformCameraUpdateNewLatLng class]]) {
+    return [GMSCameraUpdate setTarget:FGMGetCoordinateForPigeonLatLng(
+                                          ((FGMPlatformCameraUpdateNewLatLng *)update).latLng)];
+  } else if ([update isKindOfClass:[FGMPlatformCameraUpdateNewLatLngBounds class]]) {
+    FGMPlatformCameraUpdateNewLatLngBounds *typedUpdate =
+        (FGMPlatformCameraUpdateNewLatLngBounds *)update;
+    return
+        [GMSCameraUpdate fitBounds:FGMGetCoordinateBoundsForPigeonLatLngBounds(typedUpdate.bounds)
+                       withPadding:typedUpdate.padding];
+  } else if ([update isKindOfClass:[FGMPlatformCameraUpdateNewLatLngZoom class]]) {
+    FGMPlatformCameraUpdateNewLatLngZoom *typedUpdate =
+        (FGMPlatformCameraUpdateNewLatLngZoom *)update;
+    return [GMSCameraUpdate setTarget:FGMGetCoordinateForPigeonLatLng(typedUpdate.latLng)
+                                 zoom:typedUpdate.zoom];
+  } else if ([update isKindOfClass:[FGMPlatformCameraUpdateScrollBy class]]) {
+    FGMPlatformCameraUpdateScrollBy *typedUpdate = (FGMPlatformCameraUpdateScrollBy *)update;
+    return [GMSCameraUpdate scrollByX:typedUpdate.dx Y:typedUpdate.dy];
+  } else if ([update isKindOfClass:[FGMPlatformCameraUpdateZoomBy class]]) {
+    FGMPlatformCameraUpdateZoomBy *typedUpdate = (FGMPlatformCameraUpdateZoomBy *)update;
+    if (typedUpdate.focus) {
+      return [GMSCameraUpdate zoomBy:typedUpdate.amount
+                             atPoint:FGMGetCGPointForPigeonPoint(typedUpdate.focus)];
+    } else {
+      return [GMSCameraUpdate zoomBy:typedUpdate.amount];
+    }
+  } else if ([update isKindOfClass:[FGMPlatformCameraUpdateZoom class]]) {
+    if (((FGMPlatformCameraUpdateZoom *)update).out) {
+      return [GMSCameraUpdate zoomOut];
+    } else {
+      return [GMSCameraUpdate zoomIn];
+    }
+  } else if ([update isKindOfClass:[FGMPlatformCameraUpdateZoomTo class]]) {
+    return [GMSCameraUpdate zoomTo:((FGMPlatformCameraUpdateZoomTo *)update).zoom];
+  }
+  return nil;
+}
+
+UIColor *FGMGetColorForRGBA(NSInteger rgba) {
+  return [UIColor colorWithRed:((float)((rgba & 0xFF0000) >> 16)) / 255.0
+                         green:((float)((rgba & 0xFF00) >> 8)) / 255.0
+                          blue:((float)(rgba & 0xFF)) / 255.0
+                         alpha:((float)((rgba & 0xFF000000) >> 24)) / 255.0];
```

Comment:
I realise this is just moving but... technically `(CGFloat)` though in the end entirely the same thing.

Reply:
Fixed. It's technically not the same since `CGFloat` is `double` on all supported devices now, so we were pointlessly losing some precision (although probably not enough to actually matter in a color).